sdk/storage/Azure.Storage.DataMovement.Blobs/src/BlobDestinationCheckpointData.cs

```diff
@@ -40,8 +40,7 @@ internal class BlobDestinationCheckpointData : BlobCheckpointData
         /// <summary>
         /// The access tier of the destination blob.
         /// </summary>
-        public DataTransferProperty<AccessTier?> AccessTier;
-        public bool PreserveAccessTier;
+        public AccessTier? AccessTierValue;
```

Comment:
nit: Just call it `AccessTier`?

Reply:
I wanted to... but having a enum that's also called AccessTier, made it confusing. So that's why I renamed to it AccessTierValue